src/components/Item/VisualizationItem/ItemHeaderButtons.js

```diff
@@ -15,178 +20,127 @@ import {
-    [CHART, EVENT_CHART].includes(activeType)
-        ? activeStyle
-        : inactiveStyle(disabled);
-
-const mapBtnStyle = (activeType, disabled) =>
-    [MAP].includes(activeType) ? activeStyle : inactiveStyle(disabled);
-
-class VisualizationItemHeaderButtons extends Component {
-    renderInterpretationButton() {
-        const { activeFooter, onToggleFooter } = this.props;
-
-        const toggleFooterBase = activeFooter ? activeStyle : baseStyle;
-
-        const toggleFooter = {
-            ...toggleFooterBase,
-            container: {
-                ...toggleFooterBase.container,
-                ...style.toggleFooterPadding,
-                ...style.border,
-            },
-        };
-
-        return (
-            <Fragment>
-                <ItemHeaderButton
-                    style={toggleFooter.container}
-                    onClick={onToggleFooter}
-                >
-                    <MessageIcon style={toggleFooter.icon} />
-                </ItemHeaderButton>
-            </Fragment>
-        );
-    }
-
-    renderVisualizationButtons() {
-        const {
-            item,
-            visualization,
-            onSelectActiveType,
-            activeType,
-        } = this.props;
-
-        if (isSingleValue(visualization.type)) {
-            return null;
-        }
+    const onViewTable = () => {
+        onSelectActiveType(isTrackerType ? EVENT_REPORT : REPORT_TABLE);
+        handleClose();
+    };
 
-        const onViewTable = () =>
-            onSelectActiveType(
-                isTrackerDomainType(item.type) ? EVENT_REPORT : REPORT_TABLE
-            );
+    const onViewChart = () => {
+        onSelectActiveType(isTrackerType ? EVENT_CHART : CHART);
+        handleClose();
+    };
 
-        const onViewChart = () =>
-            onSelectActiveType(
-                isTrackerDomainType(item.type) ? EVENT_CHART : CHART
-            );
+    const onViewMap = () => {
+        onSelectActiveType(MAP);
+        handleClose();
+    };
 
-        const onViewMap = () => onSelectActiveType(MAP);
+    const itemHasMapView = () => hasMapView(item.type);
 
-        // disable toggle buttons
-        let disabled = false;
+    const handleMenuClick = (_, event) => setAnchorEl(event.currentTarget);
 
-        if (item.type === CHART) {
-            if (extractFavorite(item).type.match(/^YEAR_OVER_YEAR/)) {
-                disabled = true;
-            }
+    const handleInterpretationClick = () => {
+        props.onToggleFooter();
+        if (anchorEl !== null) {
+            handleClose();
         }
+    };
+
+    const handleClose = () => setAnchorEl(null);
+
+    const type = visualization.type || item.type;
+    const canViewAs =
+        !isSingleValue(props.visualization.type) &&
+        !type.match(/^YEAR_OVER_YEAR/);
+
+    const interpretationMenuLabel = props.activeFooter
+        ? i18n.t(`Show interpretations and details`)
+        : i18n.t(`View interpretations and details`);
+
```

Comment:
These messages are mixed up and one of them needs to be made to it's opposite. Right now when the interpretations are visible the menu item says 'Show interpretations and details'. This should be 'Hide interpretations and details'.

Reply:
After reading your JIRA message, I agree. Let's adjust to:
`Show interpretations and details` and `Hide interpretations and details`

Reply:
oops. Will fix